pkg/clusteragent/admission/mutate/autoinstrumentation/language_versions.go

```diff
@@ -51,18 +52,18 @@ func (l language) libInfo(ctrName, image string) libInfo {
 }
 
 // DEV: Will attempt to resolve, defaults to legacy if unable
-func (l language) libInfoWithResolver(ctrName, registry string, version string, imageResolver ImageResolver) libInfo {
-	resolvedImage, ok := imageResolver.Resolve(registry, fmt.Sprintf("dd-lib-%s-init", l), version)
-	var image string
-	if !ok {
-		image = l.libImageName(registry, version)
-	} else {
-		image = resolvedImage.FullImageRef
+func (l language) libInfoWithResolver(ctrName, registry string, version string) libInfo {
```

Comment:
I think this was called out elsewhere, but `libInfoWithResolver` doesn't accept a resolver anymore :)